### DISCUSSION TITLE
“基座模型”和“指令模型”该怎么使用？

### DISCUSSION THREAD

**Discussion by kgdxpr** [ymcui/Chinese-LLaMA-Alpaca-2#523] [Q&A]
刚接触大模型有个问题帮忙解答一下，在完整模型的列表中分了两个类型“基座模型”和“指令模型”，这两个有什么区别，在使用的时候是选择其中一个用还是两个都要用，在什么场景该用什么类型？谢谢。

**Comment by ymcui** [comment]
https://github.com/ymcui/Chinese-LLaMA-Alpaca-2?tab=readme-ov-file#模型选择指引
写的很清楚了。按需选择对应模型。
